src/style.css

```diff
@@ -0,0 +1,78 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.searchbar_container {
+  padding: 0;
+  margin-top: 20px;
+  position: relative;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+}
+
+.searchbar,
+.searchbar::placeholder {
+  height: 100%;
+  width: 100%;
+  border: none;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+[data-qa="big"] .searchbar_container {
+  height: 70px;
+}
+
+[data-qa="small"] .searchbar_container {
+  height: 42px;
+}
+
+[data-qa="big"] .searchbar {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+[data-qa="small"] .searchbar {
+  padding-left: 33px;
+  font-size: 14px;
+}
```

Comment:
Creative solution, but
<img width="819" alt="Screenshot 2023-01-24 at 19 35 10" src="https://user-images.githubusercontent.com/84275496/214366274-2651d318-1efe-4a3c-8665-022f91772de9.png">

`data` attributes are used for test, let's use instruments by their purpose